src/emailLogin/runMigration.js

```diff
@@ -0,0 +1,88 @@
+const axios = require('axios');
+const fs = require('fs');
+
+const { getDb } = require('../../db/index');
+
+const { PERSONAL_ACCESS_TOKEN, GITHUB_ORG_NAME: ISOMER_GITHUB_ORG_NAME } = process.env;
+
+const ISOMER_USERS = ['isomeradmin', 'rc-davis', 'lamkeewei', 'pallani', 'LoneRifle', 'prestonlimlianjie', 'alexanderleegs', 'lisatjide', 'kwajiehao', 'gweiying', 'seaerchin', 'isomer-demo', 'NatMaeTan', ' jacksonOGP', 'chienlinggg', 'kathleenkhy', 'joshuajunmingt', 'audreytcy', 'yanjunquek', 'chloe-opengovsg', 'shazlithebestie', 'lennardl', 'oliverli', 'taufiq'];
+
+// name of repo
+const REPO = process.argv[2];
+
+const getSiteAndContributors = async (site, dbClient) => {
+  try {
+    const { data: respData } = await axios.get(
+      `https://api.github.com/orgs/${ISOMER_GITHUB_ORG_NAME}/teams/${site}/members`,
+      {
+        headers: {
+          Authorization: `token ${PERSONAL_ACCESS_TOKEN}`,
+        },
+      },
+    );
+    if (!respData) {
+      return;
+    }
+    const contributorNames = respData
+      .map((data) => {
+        const {
+          login,
+        } = data;
+        return login;
+      })
+      .filter((name) => !ISOMER_USERS.includes(name));
+    // get user information
+    const contributorQuery = `SELECT * FROM "users" WHERE ${contributorNames.map((githubId) => `github_id='${githubId}'`).join(' OR ')};`;
+    const userData = (await dbClient.query(contributorQuery)).rows;
+    console.log(userData);
+
+    // get repo information
+    const repoQuery = `SELECT sites.id, sites.job_status, sites.site_status, repos.name FROM "sites" JOIN "repos" ON sites.id = repos.site_id WHERE repos.name='${site}';`;
+    const repoData = (await dbClient.query(repoQuery)).rows;
+    console.log(repoData);
+
+    // get list of whitelisted domains
+    const whitelistQuery = 'SELECT * FROM "whitelist" WHERE expiry IS NULL;';
+    const whitelistedDomains = (await dbClient.query(whitelistQuery)).rows.map((whitelistData) => whitelistData.email);
+    console.log(whitelistedDomains);
+
+    const repoId = repoData[0].id;
```

Comment:
RepoData.name is be the github name of a repo, which is unique - so there should only be a single item returned from the query

Reply:
this seems like an important assumption - we should write this down in code + throw an error if there are 2 identical github names rather than have it silently proceed on

Reply:
yeah that's a good point, added an error and early termination in https://github.com/isomerpages/isomer-conversion-scripts/pull/56/commits/9843d960a98dc97b6891f75231a3a57492e7a23f